File: Arbitration/MPArbitration/Model/CaseBenchmark.cs
using System.Text.Json.Serialization;
using System.ComponentModel.DataAnnotations.Schema;
using Microsoft.EntityFrameworkCore;
using System.ComponentModel.DataAnnotations;

namespace MPArbitration.Model
{
    public class CaseBenchmark
    {
        [JsonPropertyName("id")]
        public int Id { get; set; } = 0;
        
        [JsonPropertyName("extendedValue")]
        public double ExtendedValue { get; set; } = 0;

        [JsonPropertyName("isHidden")]
        public bool IsHidden { get; set; } = false;

        [JsonPropertyName("procedureCode")]
        [StringLength(20)]
        public string ProcedureCode { get; set; } = "";

        [JsonPropertyName("sortOrder")]
        public int SortOrder { get; set; } = 0;

        [JsonPropertyName("updatedBy")]
        [StringLength(60)]
        public string UpdatedBy { get; set; } = "";

        [JsonPropertyName("updatedOn")]
        public DateTime? UpdatedOn { get; set; } = null;

        [JsonPropertyName("value")]
        public double Value { get; set; } = 0;

        [JsonPropertyName("valueField")]
        [StringLength(60)]
        public string ValueField { get; set; } = "";

    }
}

File: Arbitration/MPArbitration/ClientApp/src/app/model/case-benchmark.ts
export class CaseBenchmark {
    public id = 0;
    public extendedValue = 0;
    public formLabel = '';
    public isHidden = false;
    public procedureCode = '';
    public sortOrder = 0;
    public tableLabel = '';
    public value = 0;
    public valueField = '';
    public updatedOn: Date|undefined;
    public updatedBy = '';
    
    constructor(obj?:any) {
        if(!obj)
            return;
        Object.assign(this, JSON.parse(JSON.stringify(obj)));
        if(this.updatedOn)
            this.updatedOn = new Date(obj.updatedOn);
    }
}
